pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -85,24 +85,16 @@ func NewStaticPolicyOptions(policyOptions map[string]string) (StaticPolicyOption
 			return opts, err
 		}
 
+		optValue, err := strconv.ParseBool(value)
+		if err != nil {
+			return opts, fmt.Errorf("bad value for option %q: %w", name, err)
+		}
```

Comment:
Theoretically moving this code here is not fully equivalent to the code below. In case if `name` is not among the known names strconv.ParseBool(value) should not be called.

Reply:
Yeah, it's not the same. We might never reach the `default:` case which would print the appropriate error. I'd suggest not to merge this PR

Reply:
Agreed.